### DISCUSSION TITLE
Brew runtime error

### DISCUSSION THREAD

**Discussion by jackx99** [Homebrew/discussions#1397] [Getting started]
I haven't used homebrew for quite a while. Today i tried to run some commands and this shows,
Traceback (most recent call last):
	5: from /usr/local/Homebrew/Library/Homebrew/brew.rb:31:in `<main>'
	4: from /usr/local/Homebrew/Library/Homebrew/brew.rb:31:in `require_relative'
	3: from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `<top (required)>'
	2: from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `require_relative'
	1: from /usr/local/Homebrew/Library/Homebrew/load_path.rb:9:in `<top (required)>'
/usr/local/Homebrew/Library/Homebrew/utils/gems.rb:46:in `setup_gem_environment!': unknown keyword: setup_path (ArgumentError)

**Comment by SMillerDev** [comment]
Try running `brew update-reset`

**Reply by jackx99** [reply]
I try few times to reinstall the brew and showing another error about command line tools.
xcrun: error: active developer path ("/Applications/Xcode.app/Contents/Developer") does not exist
Use `sudo xcode-select --switch path/to/Xcode.app` to specify the Xcode that you wish to use for command line developer tools, or use `xcode-select --install` to install the standalone command line developer tools.
See `man xcode-select` for more details.
Failed during: git init -q

i resolve it with https://stackoverflow.com/a/65725468/12265962
